src/AwsClientTrait.php

```diff
@@ -61,6 +61,7 @@ public function execute(CommandInterface $command)
 
     public function executeAsync(CommandInterface $command)
     {
+        $command = clone $command;
```

Comment:
Not strictly, but executing a command consumes it in the same way presigning does. Without cloning here, you can only execute a command once.